python/pyspark/ml/param/__init__.py

```diff
@@ -170,6 +170,15 @@ def toVector(value):
         raise TypeError("Could not convert %s to vector" % value)
 
     @staticmethod
+    def toMatrix(value):
+        """
+        Convert a value to ML Matrix, if possible
```

Comment:
This is not a big issue, but you can still refer the [clarification](https://spark.apache.org/docs/latest/ml-guide.html#announcement-dataframe-based-api-is-primary-api) in MLlib user guide to get the convention in MLlib.